tsengcoin-core/src/main.rs

```diff
@@ -8,17 +7,29 @@ pub mod tsengscript_interpreter;
 pub mod script_error;
 pub mod difficulty;
 pub mod hash;
+pub mod banner;
+
+use std::{error::Error, io};
+
+use banner::{BANNER};
+
 
-use std::{error::Error, env};
 
 use command::{dispatch_command};
 use commands::top_level::make_command_map;
 
 fn main() -> Result<(), Box<dyn Error>> {
+    // println!("{}", BANNER); -> Cool looking banner.
     let command_map = make_command_map();
-    let args: Vec<String> = env::args().collect();
+    loop {
```

Comment:
I think it would be best if this is not in a loop because in most cases you'll want to connect to the TsengCoin network, so having the program run 1 top level command and exit is convenient because you can just do `cargo run connect ...`. If it's in a loop, you have to type `cargo run` then type `connect ...` in separately which is a bit of a hassle. The program also starts a command loop when you join the network so it can be a little confusing having a loop running inside of another loop - when you type "quit", which loop should we exit from?

Reply:
Oh, I see. In that case quit should exit the loop when you join the network then I think. The main function should return to how you wrote it.